### ISSUE TITLE
Script indentation inconsistencies (spaces vs tabs) fails in Godot 3.2-dev

### ISSUE THREAD

**Issue by Xrayez** [bitwes/Gut#126]
**Version:** 6.8.0
**Godot version:** 3.2-dev godotengine/godot@1107799de5d487f285d8a743357fd6928351a06d (namely godotengine/godot#30849)

![spaces-vs-tabs](https://user-images.githubusercontent.com/17108460/61994604-40741880-b085-11e9-88e9-e0e9945a7f03.png)

There are a bunch of other places where this error occurs. So might be worth to make indentation consistent even before Godot 3.2 is released.

**Comment by Xrayez** [comment]
Seems like that's all:
```diff
diff --git a/addons/gut/gut.gd b/addons/gut/gut.gd
index 8f4f418..76e1fb6 100644
--- a/addons/gut/gut.gd
+++ b/addons/gut/gut.gd
@@ -581,7 +581,7 @@ func _test_the_scripts(indexes=[]):
 			_current_test = the_script.tests[i]
 
 			if((_unit_test_name != '' and _current_test.name.find(_unit_test_name) > -1) or
-			   (_unit_test_name == '')):
+				(_unit_test_name == '')):
 				p(_current_test.name, 1)
 				_new_summary.add_test(_current_test.name)
 
diff --git a/addons/gut/gut_cmdln.gd b/addons/gut/gut_cmdln.gd
index 4c71390..8888126 100644
--- a/addons/gut/gut_cmdln.gd
+++ b/addons/gut/gut_cmdln.gd
@@ -88,7 +88,7 @@ class OptionResolver:
 
 	func to_s():
 		return str("base:\n", _string_it(base_opts), "\n", \
-		           "config:\n", _string_it(config_opts), "\n", \
+				   "config:\n", _string_it(config_opts), "\n", \
 				   "cmd:\n", _string_it(cmd_opts), "\n", \
 				   "resolved:\n", _string_it(get_resolved_values()))
 
@@ -149,11 +149,11 @@ var _run_single = false
 func setup_options():
 	var opts = Optparse.new()
 	opts.set_banner(('This is the command line interface for the unit testing tool Gut.  With this ' +
-	                'interface you can run one or more test scripts from the command line.  In order ' +
-	                'for the Gut options to not clash with any other godot options, each option starts ' +
-	                'with a "g".  Also, any option that requires a value will take the form of ' +
-	                '"-g<name>=<value>".  There cannot be any spaces between the option, the "=", or ' +
-	                'inside a specified value or godot will think you are trying to run a scene.'))
+					'interface you can run one or more test scripts from the command line.  In order ' +
+					'for the Gut options to not clash with any other godot options, each option starts ' +
+					'with a "g".  Also, any option that requires a value will take the form of ' +
+					'"-g<name>=<value>".  There cannot be any spaces between the option, the "=", or ' +
+					'inside a specified value or godot will think you are trying to run a scene.'))
 	opts.add('-gtest', [], 'Comma delimited list of full paths to test scripts to run.')
 	opts.add('-gdir', [], 'Comma delimited list of directories to add tests from.')
 	opts.add('-gprefix', 'test_', 'Prefix used to find tests when specifying -gdir.  Default "[default]"')
@@ -164,11 +164,11 @@ func setup_options():
 	opts.add('-glog', 1, 'Log level.  Default [default]')
 	opts.add('-gignore_pause', false, 'Ignores any calls to gut.pause_before_teardown.')
 	opts.add('-gselect', '', ('Select a script to run initially.  The first script that ' +
-	                          'was loaded using -gtest or -gdir that contains the specified ' +
-	                          'string will be executed.  You may run others by interacting ' +
-                              'with the GUI.'))
+							'was loaded using -gtest or -gdir that contains the specified ' +
+							'string will be executed.  You may run others by interacting ' +
+							'with the GUI.'))
 	opts.add('-gunit_test_name', '', ('Name of a test to run.  Any test that contains the specified ' +
-                                 'text will be run, all others will be skipped.'))
+								'text will be run, all others will be skipped.'))
 	opts.add('-gh', false, 'Print this help, then quit')
 	opts.add('-gconfig', 'res://.gutconfig.json', 'A config file that contains configuration information.  Default is res://.gutconfig.json')
 	opts.add('-ginner_class', '', 'Only run inner classes that contain this string')
@@ -315,8 +315,8 @@ func _init():
 			quit()
 		elif(o.get_value('-gpo')):
 			print('All command line options and where they are specified.  ' +
-			      'The "final" value shows which value will actually be used ' +
-				  'based on order of precedence (default < .gutconfig < cmd line).' + "\n")
+					'The "final" value shows which value will actually be used ' +
+					'based on order of precedence (default < .gutconfig < cmd line).' + "\n")
 			print(opt_resolver.to_s_verbose())
 			quit()
 		elif(o.get_value('-gprint_gutconfig_sample')):
diff --git a/addons/gut/gut_plugin.gd b/addons/gut/gut_plugin.gd
index b7c24f1..0517c1c 100644
--- a/addons/gut/gut_plugin.gd
+++ b/addons/gut/gut_plugin.gd
@@ -2,11 +2,11 @@ tool
 extends EditorPlugin
 
 func _enter_tree():
-    # Initialization of the plugin goes here
-    # Add the new type with a name, a parent type, a script and an icon
-    add_custom_type("Gut", "Control", preload("gut.gd"), preload("icon.png"))
+	# Initialization of the plugin goes here
+	# Add the new type with a name, a parent type, a script and an icon
+	add_custom_type("Gut", "Control", preload("gut.gd"), preload("icon.png"))
 
 func _exit_tree():
-    # Clean-up of the plugin goes here
-    # Always remember to remove it from the engine when deactivated
-    remove_custom_type("Gut")
+	# Clean-up of the plugin goes here
+	# Always remember to remove it from the engine when deactivated
+	remove_custom_type("Gut")
diff --git a/addons/gut/spy.gd b/addons/gut/spy.gd
index 32ae414..5d501e4 100644
--- a/addons/gut/spy.gd
+++ b/addons/gut/spy.gd
@@ -63,7 +63,7 @@ func get_call_parameters(variant, method_name, index=-1):
 			to_return = _calls[variant][method_name][get_index]
 		else:
 			_lgr.error(str('Specified index ', index, ' is outside range of the number of registered calls:  ', call_size))
-			
+
 	return to_return
 
 func call_count(instance, method_name, parameters=null):

```

**Comment by akien-mga** [comment]
Note that for parameters split over multiple lines, the current usage will likely be supported again in the master branch when godotengine/godot#30937 is fixed, so you probably don't have to change those.

The `gut_plugin.gd` and `spy.gd` indentation fixes should be good to take in any case (@Xrayez Did the latter really trigger a syntax error, or is it just a cleanup?).

**Comment by Xrayez** [comment]
The one in `spy` seems that was trimmed by the editor so I didn't notice, the rest did trigger the error.

**Comment by bitwes** [comment]
Addressed in `godot_3_2` branch.  I'll close this when 3.2 is out of alpha and the branch is merged into master.

**Comment by bitwes** [comment]
Implemented in 6.8.1
